### DISCUSSION TITLE
Option to move tab to other windows and profiles

### DISCUSSION THREAD

**Discussion by RABINGG** [zen-browser/desktop#2425] [Ideas]
### Captchas

- [X] I have read the instructions.
- [X] I have searched existing issues and avoided creating duplicates.
- [ ] I am not filing an enhancement request.

### What happened?

Provide an option to move tab to other windows and profiles

### Reproducible?

- [ ] I have checked that this issue cannot be reproduced on Mozilla Firefox.

### Version

1.0.1-a.6 (64-bit)

### What platform are you seeing the problem on?

Windows

**Comment by linear** [comment]
<p><a href="https://linear.app/zen-browser/issue/ZEN-1809/option-to-move-tab-to-other-windows-and-profiles">ZEN-1809 Option to move tab to other windows and profiles</a></p>

**Comment by RABINGG** [comment]
@mr-cheff This is a must have feature for browser with profiles
